Spigot-Server-Patches/0556-Operator-API-Impl.patch

```diff
@@ -0,0 +1,183 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: BuildTools <unconfigured@null.spigotmc.org>
+Date: Thu, 6 Aug 2020 23:55:39 +0800
+Subject: [PATCH] Operator-API-Impl
+
+
+diff --git a/src/main/java/net/minecraft/server/PlayerList.java b/src/main/java/net/minecraft/server/PlayerList.java
+index 9382e8f79e8edec8885c629a36e230fbec50e1fb..fda45f3e4588c46d40ccedf5ef1fd1dbc2b70ae3 100644
+--- a/src/main/java/net/minecraft/server/PlayerList.java
++++ b/src/main/java/net/minecraft/server/PlayerList.java
+@@ -1011,6 +1011,19 @@ public abstract class PlayerList {
+         return this.l;
+     }
+ 
++    public void setOPLevel(GameProfile profile, int level) {
```

Comment:
Keep convention - in method below you have add**Op**, here you have set**OP**Level.